teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md

```diff
@@ -56,6 +56,43 @@ EnableAcsUsers       : False
 
 ## PARAMETERS
 
+### -Filter
+Enables you to use wildcards when specifying the Teams and ACS federation configuration settings to be returned.
+Because you can only have a single, global instance of these settings there is little reason to use the Filter parameter.
+However, if you prefer, you can use syntax similar to this to retrieve the global settings: -Identity "g*".
+
+```yaml
+Type: String
+Parameter Sets: Filter
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
```

Comment:
Same as with the filter parameter. Add to syntax block.

Reply:
Addressed